pyccl/core.py

```diff
@@ -518,18 +530,41 @@ def _build_parameters(
-                    n_s, bcm_log10Mc, bcm_etab, bcm_ks,
-                    mu_0, sigma_0, mnu_final_list, status)
+                self._params, status = lib.parameters_create_nu(Omega_c,
+                                                                Omega_b,
+                                                                Omega_k, Neff,
+                                                                w0, wa, h,
+                                                                norm_pk, n_s,
+                                                                bcm_log10Mc,
+                                                                bcm_etab,
+                                                                bcm_ks,
+                                                                mu_0,
+                                                                sigma_0, c1_mg,
+                                                                c2_mg,
+                                                                lambda_mg,
+                                                                mnu_final_list,
+                                                                status)
             else:
                 # Create ccl_parameters with modified growth arrays
-                self._params, status = lib.parameters_create_nu_vec(
-                    Omega_c, Omega_b, Omega_k, Neff,
-                    w0, wa, h, norm_pk,
-                    n_s, bcm_log10Mc, bcm_etab, bcm_ks,
-                    mu_0, sigma_0, z_mg, df_mg, mnu_final_list, status)
+                self._params, status = lib.parameters_create_nu_vec(Omega_c,
+                                                                    Omega_b,
+                                                                    Omega_k,
+                                                                    Neff,
+                                                                    w0, wa, h,
+                                                                    norm_pk,
+                                                                    n_s,
+                                                                    bcm_log10Mc,
+                                                                    bcm_etab,
+                                                                    bcm_ks,
+                                                                    mu_0,
+                                                                    sigma_0,
+                                                                    c1_mg,
+                                                                    c2_mg,
+                                                                    lambda_mg,
+                                                                    z_mg,
+                                                                    df_mg,
+                                                                    mnu_final_list,
```

Comment:
flake8 seems to be failing on github actions because of this line being too long. I'm now sure how to solve this problem really since it's already a bit horrible other than making that variable name shorter. @damonge ?

Reply:
The previous version of this line had the arguments wrapped up nicely, so I'd just copy that (check the diff)

Reply:
gave me trouble earlier but this time flake8 was nicer, it let me do what I wanted, so this is fixed.